packages/gert/install

```diff
@@ -2,12 +2,13 @@
 set -x
 set -e
 
-OS_CODENAME=$(lsb_release -cs)
```

Comment:
Codename is the attribute in `lsb_release` terms.
Distribution is very close to distributor, which is Ubuntu.

```
jss@jenkins1:~$ lsb_release --help
Usage: lsb_release [options]

Options:
  -h, --help         show this help message and exit
  -v, --version      show LSB modules this system supports
  -i, --id           show distributor ID
  -d, --description  show description of this distribution
  -r, --release      show release number of this distribution
  -c, --codename     show code name of this distribution
  -a, --all          show all of the above information
  -s, --short        show requested information in short format
```

Reply:
I was just trying to standardize on what the rest of the install scripts were doing, to make things consistent.  I don't mind whatever we name it, as long as we do it the same in each install script

Reply:
Yep, I saw that. I'm arguing that `OS_CODENAME` should be what we standardize upon.

Reply:
Ah, ok, Makes sense.  Done